packages/flac/flac.0.1.4/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+maintainer: "Romain Beauxis <toots@rastageeks.org>"
+authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+homepage: "https://github.com/savonet/ocaml-flac"
+build: [
+  ["./configure" "--prefix" prefix] {os != "macos"}
+  [
+    "./configure"
+    "CFLAGS=-I/usr/local/include"
+    "LDFLAGS=-L/usr/local/lib"
+    "OCAMLFLAGS=-ccopt -I/usr/local/include -cclib -L/usr/local/lib"
+    "--prefix"
+    prefix
+  ] {os = "macos"}
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: ["ocamlfind" "remove" "flac"]
+depends: [
+  "ocaml" {>= "4.02"}
+  "ocamlfind"
```

Comment:
```suggestion
  "ocamlfind" {build}
```
ocamlfind should probably be flagged with the `{build}` attribute as it doesn't seem to be used anywhere at runtime